.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a report to help us fix the issue
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## The problem
+
+Briefly describe the issue you are experiencing (or the feature you want to see added to Percy). Tell us what you were trying to do and what happened instead. Remember, this is _not_ a place to ask questions. For that, go to https://github.com/percy/cli/discussions/new
+
+## Reach out to Percy support
```

Comment:
I think we should remove this headline so people don't accidentally leave it in assuming that it's part of the desired issue content. I can easily see this being filled in as if it were a question being asked, such as:

```markdown
## Reach out to Percy support

already done
```

We should also move the below redirection to be first and foremost directly after the first heading. So it's the very first thing people read and don't accidentally skim over it.